fog/view/server/test-utils/src/lib.rs

```diff
@@ -227,6 +227,7 @@ impl RouterTestEnvironment {
                     sharding_strategy,
                     postgres_config: Default::default(),
                     block_query_batch_size: 2,
+                    db_polling_interval_ms: Default::default(),
```

Comment:
I wonder if this is causing the fog test failures or if those are just the flakey ones
```suggestion
                    db_polling_interval_ms: Duration::from_millis(250),
```

Reply:
Is that `db_polling_interval_ms: Default::default()` not picking up 250ms from the clap default config values?  

I could set this back to 100ms for the tests.

Reply:
No the default here would be `Duration::default()` which is probably 0